client/src/assets/scss/overrides.scss

```diff
@@ -83,4 +83,20 @@
     background-color: transparent !important;
     box-shadow: none !important;
   }
+
+  .v-btn.btn-outlined-blue {
+    background-color: white !important;
+    color: $app-blue !important;
+    border: 1px solid $app-blue !important;
+    box-shadow: none !important;
+    text-transform: none !important;
+  }
+  
+  .v-btn.btn-outlined-red {
```

Comment:
You probably don't need these. Instead, see base.scss, "button-blue" and "button-red".

Reply:
let me check